src/Service.Tests/SqlTests/GraphQLPaginationTests/MySqlGraphQLPaginationTests.cs

```diff
@@ -25,43 +25,68 @@ public static async Task SetupAsync(TestContext context)
+        [DataRow("byte_types", 0, 1, 2, 1, false, DisplayName = "Test after token for byte values.")]
+        [DataRow("byte_types", 1, "", 1, "", true, DisplayName = "Test after token for byte values for last page.")]
+        [DataRow("short_types", -32768, 1, 3, 1, false, DisplayName = "Test after token for short values.")]
+        [DataRow("short_types", 1, "", 1, "", true, DisplayName = "Test after token for short values for last page.")]
+        [DataRow("int_types", -2147483648, 1, 3, 1, false,
             DisplayName = "Test after token for int values.")]
-        [DataRow("long_types", -9223372036854775808, 9.223372036854776E+18, 3, 4,
+        [DataRow("int_types", 1, "", 1, "", true,
+            DisplayName = "Test after token for int values for last page.")]
+        [DataRow("long_types", -9223372036854775808, 1, 3, 1, false,
             DisplayName = "Test after token for long values.")]
-        [DataRow("string_types", "\"\"", "\"null\"", 1, 4,
+        [DataRow("long_types", 1, "", 1, "", true,
+            DisplayName = "Test after token for long values for last page.")]
+        [DataRow("string_types", "\"\"", "\"null\"", 1, 3, false,
             DisplayName = "Test after token for string values.")]
-        [DataRow("single_types", -3.39E38, 3.3999999521443642E+38, 3, 4,
+        [DataRow("string_types", "null", "", 3, "", true,
+            DisplayName = "Test after token for string values for last page.")]
+        [DataRow("single_types", -3.39E38, .33000001, 3, 1, false,
             DisplayName = "Test after token for single values.")]
-        [DataRow("float_types", -1.7E308, 1.7E308, 3, 4,
+        [DataRow("single_types", .33, "", 1, "", true,
+            DisplayName = "Test after token for single values for last page.")]
+        [DataRow("float_types", -1.7E308, .33, 3, 1, false,
             DisplayName = "Test after token for float values.")]
-        [DataRow("decimal_types", -9.292929, 0.333333, 2, 1,
+        [DataRow("float_types", .33, "", 1, "", true,
+            DisplayName = "Test after token for float values for last page.")]
+        [DataRow("decimal_types", -9.292929, 0.0000000000000292929, 2, 4, false,
             DisplayName = "Test after token for decimal values.")]
-        [DataRow("boolean_types", "false", "true", 2, 4,
+        [DataRow("decimal_types", 0.333333, "", 1, "", true,
+            DisplayName = "Test after token for decimal values for last page.")]
+        [DataRow("boolean_types", "false", "true", 2, 3, false,
             DisplayName = "Test after token for boolean values.")]
+        [DataRow("boolean_types", "true", "", 3, "", true,
+            DisplayName = "Test after token for boolean values for last page.")]
         [DataRow("datetime_types", "\"1753-01-01T00:00:00.000\"",
-            "\"9999-12-31 23:59:59.000000\"", 3, 4,
+            "\"1999-01-08T10:23:54\"", 3, 1, false,
```

Comment:
nit: please open issue to track fixing this (most likely) bug. When datetime was "\"9999-12-31 23:59:59.000000\"" the test fails locally for us (UTC-7), but not in UTC+5:30

Reply:
I'm assuming that's why this number was changed

Reply:
I adjusted these just to be sure we'd land on either the last page or before the last page. I did not notice the bug youre referencing, but can take a look and open an issue once I understand it.